docs/tutorials/tutorial1.md

```diff
@@ -15,16 +17,16 @@ These tools already exist within the Janis Tool Registry, you can see their docu
 
 ## Preparation
 
-To prepare for this tutorial, we're going to create a folder and download some data:
+To prepare for this tutorial, we're going to need to download some data first:
 
 ```bash
-mkdir janis-tutorials && cd janis-tutorials
+cd ~/janis/janis-tutorials
 
 # If WGET is installed
-wget -q -O- "https://github.com/PMCC-BioinformaticsCore/janis-workshops/raw/master/janis-data.tar" | tar -xz
+wget -q -O- "https://github.com/PMCC-BioinformaticsCore/janis-workshops/raw/master/janis-data.tar" | tar -x
```

Comment:
I think the `z` needs to be there only if the file is gzipped. Removing it the command succeeded, although with warnings (will create an issue about the warnings).